pom.xml

```diff
@@ -3478,16 +3150,6 @@
         <artifactId>poi-ooxml</artifactId>
         <version>${version.org.apache.poi}</version>
       </dependency>
-      <dependency>
-        <groupId>org.apache.poi</groupId>
-        <artifactId>poi-ooxml-schemas</artifactId>
-        <version>${version.org.apache.poi}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.apache.poi</groupId>
-        <artifactId>poi-scratchpad</artifactId>
-        <version>${version.org.apache.poi}</version>
-      </dependency>
```

Comment:
we need  to ask @ge0ffrey @manstis if these managed deps declaration are ok to remove